### DISCUSSION TITLE
PM & CM Meeting - Friday 18/10/2024

### DISCUSSION THREAD

**Discussion by EmmanetD** [clentafrica/programs#6] [Polls]
Hi team @clentafrica/wg-programs @clentafrica/wg-community-managers @yigakpoa 

I hope everyone is doing well. We should schedule a brief meeting for **tomorrow, Friday, October 18th, 2024.** 

The purpose of the meeting is to get to familiarize with each other as a team, discuss operational strategies to enhance our effectiveness, review progress and tasks, and more.

I have created a poll that I hope will help us arrive at a suitable time for everyone. 

Thank you for your prompt response and I look forward to meeting with all of you **tomorrow**. 

Regards,
Damilare. 
Team Lead, PM team

**Comment by EmmanetD** [comment]
Hello team @clentafrica/wg-programs @clentafrica/wg-community-managers , only one member voted already. Can the rest of us respond so we can finalize on the possibility of holding a meeting today. Let's not forget that there is a live GitHub tutorial tomorrow.
Thank you so much.

**Comment by ghost** [comment]
Hi Damilare and Team, I hope you're all doing well. I just wanted to check if the meeting scheduled for today, Friday, October 18th, is still on.
Looking forward to your confirmation!

**Comment by EmmanetD** [comment]
Hi @Muasya0254 , @clentafrica/wg-programs , and @clentafrica/wg-community-managers , 

We haven't received enough votes yet, with only one vote recorded so far. The voting results would have helped us determine a suitable time for the meeting.

**There's a live GitHub tutorial happening tomorrow. You can find the details, including the meeting link, here:https://github.com/clentafrica/hr-people-operations/discussions/14 .**

We'll be meeting next week, team. Let's keep that in mind. 

Thank you!
